src/renderer/gdi/gdirenderer.hpp

```diff
@@ -90,8 +90,11 @@ namespace Microsoft::Console::Render
         HFONT _hfont;
         TEXTMETRICW _tmFontMetrics;
 
-        static const size_t s_cPolyTextCache = 80;
-        POLYTEXTW _pPolyText[s_cPolyTextCache];
+        static constexpr size_t s_cPolyTextCache = 80;
+        std::array<POLYTEXTW, s_cPolyTextCache> _polyText;

```

Comment:
I figured that all those containers have exact same fixed size. Using `std::array` is more reasonable